### PR TITLE
Sync with Vox Pupuli

### PR DESCRIPTION
**Pull request by smortex**
In this branch we include what is required to validate this two PRs:

* https://github.com/voxpupuli/modulesync_config/pull/789
  (Synced from this branch)
* https://github.com/voxpupuli/voxpupuli-release/pull/44
  (Fetch the gem form the WIP branch in git rather than Rubygems)

### CODE REVIEW

**Review by neomilium — APPROVED** [review]

File: config_defaults.yml
Comment (smortex) [inline config_defaults.yml]: I'll wait for a release and update the PR accordingly before merging.

**Pull request merged**
